tests/dem/insertion_volume_1.cc

```diff
@@ -53,17 +53,13 @@ test()
   DEMSolverParameters<dim> dem_parameters;
 
   // Defining simulation general parameters
-  dem_parameters.insertion_info.x_min                                = -0.05;
-  dem_parameters.insertion_info.y_min                                = -0.05;
-  dem_parameters.insertion_info.z_min                                = -0.05;
-  dem_parameters.insertion_info.x_max                                = 0.05;
-  dem_parameters.insertion_info.y_max                                = 0.05;
-  dem_parameters.insertion_info.z_max                                = 0.05;
-  dem_parameters.insertion_info.axis_0                               = 0;
-  dem_parameters.insertion_info.axis_1                               = 1;
-  dem_parameters.insertion_info.axis_2                               = 2;
-  dem_parameters.insertion_info.inserted_this_step                   = 10;
-  dem_parameters.insertion_info.distance_threshold                   = 2;
+  dem_parameters.insertion_info.insertion_box_point_1 = {-0.05, -0.05, -0.05};
+  dem_parameters.insertion_info.insertion_box_point_2 = {0.05, 0.05, 0.05};
+  dem_parameters.insertion_info.axis_0                = 0;
+  dem_parameters.insertion_info.axis_1                = 1;
+  dem_parameters.insertion_info.axis_2                = 2;
```

Comment:
Could the axis not be like a std::vector instead of three parameters ?

Reply:
In the next pr together with the velocity and omega?